components/frontend-react/src/API/Users.js

```diff
@@ -0,0 +1,55 @@
+import axios from 'axios';
+
+//Log in the user
+const verifyUserCredentials = (action) => {
+  let formData = new FormData();
+  formData.set('user_name', action.creds.username);
+  formData.set('password', action.creds.password);
+  return axios.post(
+    `${process.env.REACT_APP_API_POST_LOGIN}`,
+    formData,    
+    {headers: {'Content-Type': 'multipart/form-data'}}
+  );
+}
+
+const addNewUser = (action) => {
```

Comment:
Doesn't `addNewUser` need a token?

Reply:
Milindu, it's working without token being required. Actually, I took reference from Angular UI and found out that token was not required

Reply:
@vivonk Is this the expected behaviour?

Reply:
Milindu, what I understood is that the first time user won't  be able to access token as it is generated only when user gets logged in. So, in order to get registered he needs to submit the form for getting approved by the admin.